.github/workflows/check_pr_plugin_cleanup_aws_loadbalancers.yml

```diff
@@ -30,19 +30,20 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ~/.cache/pip
-          key: $\{\{runner.os}}-pip-$\{\{hashFiles('setup.py')}}
+          key: $\{\{runner.os}}-pip-$\{\{hashFiles('pyproject.toml')}}
           restore-keys: |
             $\{\{ runner.os }}-pip-
 
       - name: Install Dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install --upgrade --editable resotolib/
-          pip install tox wheel flake8
+          pip install poetry nox
 
       - name: Run tests
         working-directory: ./plugins/cleanup_aws_loadbalancers
-        run: tox
+        run: |
+          nox
+          nox -rs coverage_ci
```

Comment:
Instead of 2 separate runs - can we define the list of targets that should be executed?
Same is true for all other GitHub actions.